hack/istio-mtls-meshwide.yaml

```diff
@@ -0,0 +1,25 @@
+##########
+# This enables mTLS for the whole istio installation.
```

Comment:
istio -> Istio

Reply:
also, it's not for Istio itself, right? Istio installation might be misleading

Reply:
Comment approached, phrase reformulated :)